R/age-pyramid.R

```diff
@@ -1,22 +1,45 @@
 #' Plot an age group
 #'
+#' @param data a data frame containing an age group column and a bivariate column 
+#'   defining data to create either sides of the pyramid (defaults to "sex")
```

Comment:
this no longer the case, or? Change to just a data frame

Reply:
good point! I'll change the wording

Reply:
done in 3715f96